src/main.rs

```diff
@@ -1,3 +1,73 @@
+#[macro_use]
+extern crate clap;
+
 fn main() {
-    println!("Hello, world!");
+    let app = clap_app!(myapp =>
+        (version: "0.1.0")
+        (author: "Bike Barn <https://github.com/bike-barn/hermit>")
+        (about: "A home directory configuration management assistant.")
+        (@subcommand add =>
+            (about: "Add files to your hermit shell")
+            (usage: "hermit add [[-r|--redact=] <filepattern>…]")
```

Comment:
Ah. Right. Options aren't being enforced through subcommand arguments yet, but I'll update the usage to reflect that.